### DISCUSSION TITLE
Show RTE toolbar and toolbar in the same container

### DISCUSSION THREAD

**Discussion by mamal72** [GrapesJS/grapesjs#3572] [Q&A]
Hey.

I'm trying to find a way to merge RTE toolbar and the normal toolbar. It's basically about rendering them both in the same container. So when I click on a text node, both of the toolbars will be rendered there in the same container and when I select the other non-text nodes, only the normal toolbar will be rendered inside that container. So I'll have some markup like this:

```html
<div class="main-container">
  <div class="toolbar">
  </div>

  <div class="rte-toolbar"> <!-- optional, will not be rendered on non-text nodes -->
  </div>
</div>
```

Is there any way to achieve this with some configuration change? If it's not, is there any other way with some minimal changes to do so?

**Comment by artf** [comment]
No, unfortunately, the rendering of toolbars is not yet customizable.
